housing_cooperative_base/models/lease.py

```diff
@@ -106,6 +116,22 @@ def _compute_lease_end(self):
             else:
                 lease.end = lease.expected_end
 
+    @api.multi
+    @api.depends("start", "end")
+    def _compute_state(self):
+        today = fields.Date.today()
+        if self.start and self.end:
+            if today < self.start:
+                self.state = "draft"
+            elif self.start <= today <= self.end:
+                self.state = "ongoing"
+            elif self.end < today:
+                self.state = "done"
+            else:
+                False
+        else:
```

Comment:
What happens if start *or* end is False?

Reply:
They are required, so this should not happen. But I think `if lease.start and lease.end:` handles that case?